### PR TITLE
Support multi-byte string function variants

### PR DESCRIPTION
**Pull request by staabm**

### CODE REVIEW

File: src/Type/Php/SubstrDynamicReturnTypeExtension.php [hunk 1]
```diff
@@ -25,7 +26,7 @@ class SubstrDynamicReturnTypeExtension implements DynamicFunctionReturnTypeExten
 
 	public function isFunctionSupported(FunctionReflection $functionReflection): bool
 	{
-		return $functionReflection->getName() === 'substr';
+		return in_array($functionReflection->getName(), ['substr', 'mb_substr'], true);
```
Comment (rvanvelzen) [inline src/Type/Php/SubstrDynamicReturnTypeExtension.php:30]: The `substr()` calls below should be changed to use the correct function as well.

**Comment by rvanvelzen** [comment]
Some tests with multibyte characters would be good as well 👍

File: src/Type/Php/SubstrDynamicReturnTypeExtension.php (hunk 1 above)
Comment (staabm) [reply src/Type/Php/SubstrDynamicReturnTypeExtension.php:30]: great catch, thx.

**Comment by ondrejmirtes** [comment]
Thank you.

**Pull request merged**